modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -17,23 +17,24 @@ permissions.
 
 . Navigate in the web console to the *Catalog → Operator Hub* page.
 
-. Click *Show Community Operators*.
+. Choose the Operator you want to install from the list of available Operators (*etcd* for this example), and click *Install*. A pop-up window appears, warning you that you are installing a Community Operator.
 +
 [NOTE]
 ====
 This Operator will be available under the *Red Hat* category in the GA release.
 ====
+. Select *Continue* to acknowledge the Show Community Operator warning. The Install page appears.
 
-. Choose the Operator you want to install from the list of available Operators (*etcd* for this example), and click *Install*.
+. Read the information about the Operator and click *Install*.
 
-. On the *Create Operator Subscription* page, select *All namespaces...* 
-under Installation Mode, choose an Update Channel (if more than one is available),
-And select *Automatic* or *Manual* Approval Strategy, as described earlier.
+. On the *Create Operator Subscription* page, select:
+.. *A specific namespace on the cluster* under Installation Mode to choose a namespace in which the Operator will be available (*default* installs to all namespaces)
+.. An *Update Channel* (if more than one is available)
+.. *Automatic* or *Manual* Approval Strategy, as described earlier
 
-. Select the *Subscribe* button to make the Operator available to all
-users and projects that use this {product-title} cluster.
+. Select the *Subscribe* button to make the Operator available to the selected namespaces on this {product-title} cluster.
 
-. On the *Catalog → Installed Operators* page, verify that the *etcd*
+. Select *Catalog → Installed Operators* page to verify that the *etcd*
```

Comment:
If you're going to have "page" afterwards, then you need "the" in front still. So either:

* Select the *Catalog → Installed Operators* page to...
* Select *Catalog → Installed Operators* to...

Reply:
Yup. The "page" was was a mistake. Fixed.